scripts/freeze-network-node.js

```diff
@@ -0,0 +1,59 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+/**
+ * @notice this scripts is mainly designed to freeze the local network node to prepare for network migration (mono to mod)
+ */
+const {
+  FreezeTransaction,
+  Client,
+  Timestamp,
+  FreezeType,
+} = require('@hashgraph/sdk');
+const { OPERATOR_ID_A, OPERATOR_KEY_A } = require('../utils/constants');
+
+async function main() {
+  try {
+    const genesisClient = Client.forNetwork({
```

Comment:
nit: add a comment to say assumes a single node network.
This won't work in multinode case. 
No change needed but worth noting